### ISSUE TITLE
Run Simple example error 

### ISSUE THREAD

**Issue by Jayuda** [kevinzg/facebook-scraper#912]
hello, i was try an example code : 
```
from facebook_scraper import get_posts
for post in get_posts('nintendo', pages=1):
    print(post['text'][:50])
```

But show error : 

requests.exceptions.ConnectionError: HTTPSConnectionPool(host='m.facebook.com', port=443): Max retries exceeded with url: /nintendo/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f41a7cb1340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

**Comment by EvaPJ11** [comment]
hey there, I have the same issue with the example code, the error is: 
WARNING:facebook_scraper.page_iterators:No raw posts (<article> elements) were found in this page.

I am using python in google colab. Being very new to this scraper, I can only hope that I will manage to scrape sth with time

**Comment by neon-ninja** [comment]
I don't think this will work from Colab. Most users of Facebook don't connect to Facebook from Colab, so you're pretty obviously scraping
